query/promql/promql.go

```diff
@@ -3,8 +3,6 @@
 // Package promql implements a promql parser to build flux query specifications from promql.
 package promql
 
-//lint:file-ignore SA6001 Ignore all unused code, it's generated
```

Comment:
I think you need to find some way to add this lint line.

Reply:
Thank you